react/FetchMore.js

```diff
@@ -22,10 +24,72 @@ const FetchMore = () => {
     searchQuery
   )
 
-  const changeState = () =>
-    dispatch({ type: 'SET_FETCHING_MORE', args: { isFetchingMore: true } })
+  const fetchMore = path(['fetchMore'], searchQuery)
+  const fetchMoreLocked = useRef(false)
+  const { setQuery } = useRuntime()
 
-  const isShowMore = pagination === 'show-more'
+  const pageRef = useRef(page)
+  const toRef = useRef(page * maxItemsPerPage - 1)
```

Comment:
I think you can omit `ref` here; maybe something like `currentPage` and `nextPage`? or something of the sort, if `to` means something else

Reply:
maybe `lastVisibleItem` or something